### PR TITLE
Better dev data

### PR DESCRIPTION
**Pull request by blfpd**
Screen Shot 2020-02-05 at 15 23 40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The Zoo also provides analytics about how dependency usage and its versions evol
 ### Initial setup
 
 - Create a database: `$ make migrate`
-- Create a superuser: `$ make superuser`
+- Create a superuser and fake data: `$ make fake`
```
Comment (aexvir) [inline README.md:35]: So, from what I see `make superuser` will still work, it's just that if not used we'll create one with `admin:password` credentials, right?

**Comment by aexvir** [comment]
As I mentioned in the #236 PR, what do you think about:
> To make it more useful I'd probably run sync_repos first, and pick up a random repo from Repositories.objects.all()

when creating services?

File: README.md (hunk 1 above)
Comment (blfpd) [reply README.md:35]: That’s the idea.

Comment (aexvir) [reply README.md:35]: Great

**Comment by aexvir** [comment]
> As I mentioned in the #236 PR, what do you think about:
> 
> > To make it more useful I'd probably run sync_repos first, and pick up a random repo from Repositories.objects.all()
> 
> when creating services?

Alright, so this shouldn't be a reason to block this PR, we'll discuss possible approaches for filling meaningful repo data

**Review by aexvir — APPROVED** [review]

File: zoo/factories.py [hunk 2]
```diff
@@ -42,9 +50,14 @@ class Meta:
...
 
     owner = Faker("user_name")
-    name = Faker("domain_word")
-    impact = choice(["profit", "customers", "customers"])
-    repository = SubFactory(RepositoryFactory)
+    name = Faker("slug")
+    status = LazyFunction(lambda: choice([item.value for item in Status] + [None]))
+    impact = LazyFunction(lambda: choice([item.value for item in Impact] + [None]))
+    repository = SubFactory(
+        RepositoryFactory, owner=SelfAttribute("..owner"), name=SelfAttribute("..name")
```
Comment (blfpd) [inline zoo/factories.py:57]: For now we’ll just create new repositories according to owner name and service name.

**Pull request merged**